PC/pyconfig.h

```diff
@@ -269,11 +269,11 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
                         file in their Makefile (other compilers are
                         generally taken care of by distutils.) */
 #                       if defined(_DEBUG)
-#                               pragma comment(lib,"python3_10_d.lib")
+#                               pragma comment(lib,"python3.10_d.lib")
```

Comment:
Wouldn’t it be better to consistently use `_` everywhere? Then no one has to think which to use.

Reply:
Other platforms already use dots in the equivalent filename. We decided on consistency there (though if I can't fix the crashes, we may roll back this part entirely and just change packaging stuff).